packages/fuselage/src/components/Sidebar/SidebarBanner.tsx

```diff
@@ -42,6 +44,7 @@ export const SidebarBanner = ({
       )}
       {children}
     </div>
+    {addon}
```

Comment:
what happens if you add an `addon` **and** `onClose` action?
is that situation wanted or should we prevent it?

Reply:
My intention was to not affect existing functionality. Here's an example with the `onClose`:
![Screen Shot 2023-10-13 at 12 36 46](https://github.com/RocketChat/fuselage/assets/6494543/58de7562-42d0-4426-8025-288cdfaf023e)